tests/bluetooth/iso/bsim/main.c

```diff
@@ -0,0 +1,452 @@
+					argv_copy[i + 2]);
+				return -EINVAL;
+			}
+		} else if (strcmp(argv_copy[i + 1], "iso_packets_limit_min") == 0) {
+			if (is_number(argv_copy[i + 2])) {
+				packets_limit_min[ISO] = strtol(argv_copy[i + 2], NULL, 10);
+				LOG_INF("Var set: %d", packets_limit_min[ISO]);
+			} else {
+				LOG_ERR("Argument: %s must be a number [%s]", argv_copy[i],
+					argv_copy[i + 2]);
+				return -EINVAL;
+			}
+		}
+	}
+
+	return 0;
+}
+
+static int modules_init(void)
+{
+	int err;
+
+	if (role == PERIPHERAL_SINK) {
+		TEST_START("ACL peripheral and ISO Broadcast Sink");
+
+		err = iso_broadcast_sink_init(sim_iso_recv_main_cb);
+		if (err) {
+			LOG_ERR("iso_broadcaster_sink_init failed (err %d)", err);
+			return err;
+		}
+
+		err = acl_peripheral_init(sim_acl_recv_main_cb);
+		if (err) {
+			LOG_ERR("central_init failed (err %d)", err);
+			return err;
+		}
+
+	} else if (role == CENTRAL_SOURCE) {
+		TEST_START("ACL central and ISO Broadcast source");
+
+		err = iso_broadcast_src_init();
+		if (err) {
+			LOG_ERR("iso_broadcast_src_init failed (err %d)", err);
+			return err;
+		}
+
+		err = acl_central_init();
+		if (err) {
+			LOG_ERR("central_init failed (err %d)", err);
+			return err;
+		}
+
+	} else {
+		LOG_ERR("Role not set");
+		return -EINVAL;
+	}
+
+	return 0;
+}
+
+static int modules_start(void)
+{
+	int err;
+
+	if (role == PERIPHERAL_SINK) {
+		TEST_START("ACL peripheral + ISO Broadcast Sink");
+
+		err = cmd_send(2, "acl_peripheral", "start");
+		if (err) {
+			return err;
+		}
+
+		err = cmd_send(2, "iso_brcast_sink", "start");
+		if (err) {
+			return err;
+		}
+
+	} else if (role == CENTRAL_SOURCE) {
+		TEST_START("ACL central + ISO Broadcast source");
+
+		err = cmd_send(2, "acl_central", "start");
+		if (err) {
+			return err;
+		}
+
+		err = cmd_send(2, "iso_brcast_src", "start");
+		if (err) {
+			return err;
+		}
+
+	} else {
+		return -EINVAL;
+	}
+
+	return 0;
+}
+
+void test_main(void)
+{
+	int err;
+
+	report[ACL].passed = false;
+	report[ISO].passed = false;
+	report[ACL].fail_percentage = 100;
+	report[ISO].fail_percentage = 100;
+
+	int shells = shell_backend_count_get();
+
+	if (shells != 1) {
+		TEST_FAIL("Only one shell accepted. Found: %d", shells);
+	}
+
+	err = test_params_set();
+	if (err) {
+		TEST_FAIL("test_params_set failed (err %d)", err);
+	}
+
+	err = bt_enable(NULL);
+	if (err) {
+		TEST_FAIL("Bluetooth enable failed (err %d)", err);
+	}
+
+	err = modules_init();
+	if (err) {
+		TEST_FAIL("Modules init failed (err %d)", err);
+	}
+
+	err = modules_configure();
+	if (err) {
+		TEST_FAIL("Configure failed (err %d)", err);
+	}
+
+	err = modules_start();
+	if (err) {
+		TEST_FAIL("Modules start failed (err %d)", err);
+	}
+
+	if (role == CENTRAL_SOURCE) {
+		/* There are no test cases for this role.*/
+		TEST_PASS("Central/source passed");
+	}
+}
+
+static void test_delete(void)
+{
+	if (role == CENTRAL_SOURCE) {
+		return;
+	}
+
+	const char text_color[][sizeof(COLOR_GREEN)] = {COLOR_GREEN, COLOR_RED};
```

Comment:
Instead of using `color` have you considered switching between `LOG_INF`, `LOG_WRN` and `LOG_ERR`?

Reply:
Yes. I would like to see this output regarding of if the test passes or not. Hence the color.